### DISCUSSION TITLE
keypassxc browser extension does not fill password

### DISCUSSION THREAD

**Discussion by jesinj22** [keepassxreboot/keepassxc#11552] [Q&A] [answered]
I am trying to use KeePassXC to fill in the password for a website under https://portal.azure.com/. We use Bastion to log in to Azure VMs.

When I go to the Bastion login page and click on the KeePassXC icon in the 'username' field, it successfully shows all the passwords saved for https://portal.azure.com/. However, when I select an entry, it only fills in the username and not the password.

Strangely, this issue only occurs the first time I visit the page. If I navigate away from the VM login page and return, it works as expected."

this is how it looks
![image](thttps://github.com/user-attachments/assets/02ba0a6d-6eb6-4337-9bb8-e0524ee5fef3)

**Comment by varjolintu** [comment, marked as answer]
You should create an issue to https://github.com/keepassxreboot/keepassxc-browser/issues instead of a new discussion thread.

**Reply by jesinj22** [reply]
Thanks, This is my first day with KeypassXC so just wanted to make sure I am not making any mistakes.
